vsintegration/src/FSharp.Editor/InlineHints/InlineHints.fs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
```

Comment:
"Inline Hints" is a slightly odd word given the other meanings of "inline" in the codebase.  Not sure there's a better word though

Where is `IFSharpInlineHintsService` defined?

Reply:
InlineHints is the name of the feature in Roslyn (and a future VSCode too). `IFSharpInlineHintsService` is defined in Roslyn itself, we have a shim layer there between their feature layer and the accessible API for us. It lets us get access to their feature layers but also forces them to handle breaking changes (so we don't ship with a feature broken because they changed an API on us).